README.md

```diff
@@ -40,9 +40,11 @@ This will use a suitable backend for your platform or you can choose one, for ex
 See `linuxkit run --help`.
 
 Additional, platform specific information is available for:
-- [macOS](docs/mac.md)
-- [Google Cloud](docs/gcp.md)
-- [Azure](docs/azure.md)
+- [macOS/HyperKit](docs/platform-hyperkit.md)
```

Comment:
was considering it, but I often read docs in a terminal and having another directory for a few files seems a bit too much given the current number of files we have